documentation/src/main/docs/faq.adoc

```diff
@@ -368,57 +372,22 @@ include::../../../src/test/java/snippets/UniBlockingTest.java[tags=code]
 ----
 
 Create an `Uni` that will supply the item using a blocking call, here the `invokeRemoteServiceUsingBlockingIO` method.
-To avoid blocking the subscriber, use `runSubscriptionOn` which switch the thread and so call `invokeRemoteServiceUsingBlockingIO` on another thread.
```

Comment:
I thought `runSubscriptionOn` was the new name?

Reply:
Yes, rebase issue - fixed.